modules/keyframes.js

```diff
@@ -27,6 +27,7 @@ if (isAnimationSupported) {
 // Simple animation helper that injects CSS into a style object containing the
 // keyframes, and returns a string with the generated animation name.
 var keyframes = function (
+  component: any,
```

Comment:
Good shout, I read your first sentence and then immediately thought about what you are suggesting! :smile: